src/mpc.c

```diff
@@ -650,7 +650,7 @@ static Obj MPC_2MPFR (Obj self, Obj fl, Obj fr)
 
 static Obj ROOTPOLY_MPC (Obj self, Obj coeffs, Obj precision)
 {
-  int cpoly_MPC(int, mpc_t *, mpc_t *, mp_prec_t);
+  int cpoly_MPC(int, mpc_t *, mpc_t *, int);
```

Comment:
In addition I would suggest to avoid function declarations like this inside a function. Instead, I'd move it to a new file `mpc_poly.h` (similar to `cxsc_poly.h`) which then can be included by both `mpc.c` and `mp_poly.C`. 

Of course this is not a criticism of this PR nor does it need to be addressed in this PR; I merely was "inspired" by reviewing this code.